external/deep-object-reid/requirements.txt

```diff
@@ -1,4 +1,4 @@
 nncf @ git+https://github.com/openvinotoolkit/nncf@464244204fc2c5e80c8164c17d8d266ccae50062#egg=nncf
 openvino==2022.1.0.dev20220302
 openvino-dev==2022.1.0.dev20220302
-openmodelzoo-modelapi @ git+https://github.com/openvinotoolkit/open_model_zoo/@releases/2021/SCMVP#egg=openmodelzoo-modelapi&subdirectory=demos/common/python
\ No newline at end of file
+openmodelzoo-modelapi @ git+https://github.com/openvinotoolkit/open_model_zoo/@master#egg=openmodelzoo-modelapi&subdirectory=demos/common/python
```

Comment:
ote-related stuff has been moved to training_extensions, see external/deep-object-reid